cmd/ocm/login/cmd.go

```diff
@@ -144,8 +147,16 @@ func run(cmd *cobra.Command, argv []string) error {
 	haveSecret := args.clientID != "" && args.clientSecret != ""
 	haveToken := args.token != ""
 	if !havePassword && !haveSecret && !haveToken {
-		return fmt.Errorf("In order to log in it is mandatory to use '--token', '--user' and " +
-			"'--password', or '--client-id' and '--client-secret'.")
+		// Allow bare `ocm login` to suggest the token page without extra noise.
+		fmt.Fprintf(
+			os.Stderr,
+			"In order to log in it is mandatory to use '--token', '--user' and "+
+				"'--password', or '--client-id' and '--client-secret'.\n"+
+				"You can obtain a token at: %s .\n"+
```

Comment:
Space before period is awkward but deliberate.
Not all terminal allow direct clicking of URLs, not all users know how to use that (e.g. Ctrl+click in gnome-terminal, or right click menu). The space makes it easier to grab the URL with regular mouse selection or double-click word selection.

Could trop the period altogether, but with the 2 other sentences ending in a period felt that would look awkward too.